src/main/java/seedu/address/model/poc/UniquePocList.java

```diff
@@ -0,0 +1,138 @@
+package seedu.address.model.poc;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Iterator;
+import java.util.List;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import seedu.address.model.poc.exceptions.DuplicatePocException;
+import seedu.address.model.poc.exceptions.PocNotFoundException;
+
+/**
+ * A list of companies that enforces uniqueness between its elements and does not allow nulls.
+ * A Poc is considered unique by comparing using {@code Poc#isSamePoc(Poc)}.
+ * As such, adding and updating of companies uses Poc#isSamePoc(Poc) for equality to ensure that the
```

Comment:
"companies" should be changed to POCs instead. This is a recurring issue throughout the file.